### PR TITLE
Section1Part8(and9)RestoreGaremokosPR

### PR DESCRIPTION
**Pull request by andyjohnson**
Should have all changes from @garemoko's PR

### CODE REVIEW

**Comment by andyjohnson** [comment]
Sorry about that should have everything from https://github.com/andyjohnson/xAPI-Spec/pull/35 from @garemoko about https://github.com/adlnet/xAPI-Spec/pull/713

**Comment by garemoko** [comment]
+1

File: xAPI.md [hunk 1]
```diff
@@ -533,13 +533,32 @@ __Verb__: Defines the action being done by the Actor within the Activity within
...
 
-### 8.3 Communities of Practice
+Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
+those listed in this specification can be implemented and co-exist with the Resources defined in this specification.
+
+<a name="COPs" />
+
+## 9.0 Profiles and Communities of Practice
+
+xAPI strictly defines the structure of statements communicated between the client and an LRS but is very flexible as to the content of that statement structure. For example, the specification requires that all statements have a verb property, but does not restrict the values of that property; any verb can be used. This flexibility enables Tin Can to be used in any context, including future use cases not envisaged by the specification authors.
```
Comment (ljwolford) [inline xAPI.md]: xAPI instead of Tin Can

Comment (andyjohnson) [inline xAPI.md]: LOL, ill fix

Comment (garemoko) [inline xAPI.md]: Oops ;-)

**Comment by garemoko** [comment]
In case there's any doubt, I'm still +1 after the latest commit.

**Pull request merged**